packages/core/src/StorageHelper/LocalStorage.ts

```
@@ -5,11 +5,11 @@ import { PlatformNotSupportedError } from '../Errors';
 import { AuthStorage } from '../types';
 
 export class LocalStorageClass implements AuthStorage {
-	storage: Storage;
+	storage: Storage | undefined;
 
 	constructor() {
 		try {
-			this.storage = window.localStorage;
+			this.storage = window?.localStorage ?? undefined;
 		} catch (error) {}
```

Nit but instead of swallowing errors, could we just assert window is available? (same for other managers that use it)

```
if (typeof window !== "undefined") {
    this.storage = window?.localStorage;
}
```